vcap_services.json.enc

```diff
@@ -0,0 +1,2 @@
+p���'+=/�c�L���>�����f͠W�Fʎ���?1ֱ�_���#oހ�̔��aJzjxʝ�7�������I�[�d3� [?^�5�b���R?FR@d���� 9M�uaO�x�@a��;����=��������8q�@��I���+�Y��)�L�:���u����
```

Comment:
Even encrypted this file doesn't belong in openwhisk. Others can't run these tests with this file.